corgi/tasks/management/commands/generatemanifests.py

```diff
@@ -13,17 +14,12 @@ def add_arguments(self, parser: CommandParser) -> None:
             dest="stream",
             help="Update the manifest for the named product stream",
         )
-        parser.add_argument(
-            "-f",
-            "--skip-fixups",
-            action="store_false",
-            help="Skip applying manifest fixups",
-        )
 
     def handle(self, *args, **options) -> None:
         if options["stream"]:
+            ps = ProductStream.objects.get(name=options["stream"])
             self.stdout.write(self.style.SUCCESS(f"Updating manifest for {options['stream']}"))
-            cpu_update_ps_manifest(options["stream"], fixup=options["skip_fixups"])
+            cpu_update_ps_manifest(ps.name, ps.external_name)
```

Comment:
for my own understanding - why do we no longer pass in fixup ?

Reply:
We still use some of the hardcoded values from the fixups, but they are now integrated into the ProductModel.cpes property, not just part of SBOM generation. We don't have the option to skip them anymore.